src/rabbit_cli.erl

```diff
@@ -125,9 +135,22 @@ main(ParseFun, DoFun, UsageMod) ->
             rabbit_misc:quit(?EX_SOFTWARE)
     end.
 
+start_distribution_anon(0, LastError) ->
+    {error, LastError};
+start_distribution_anon(TriesLeft, _) ->
+    NameCandidate = list_to_atom(rabbit_misc:format("rabbitmq-cli-~2..0b", [rabbit_misc:random(100)])),
```

Comment:
Then every sequence of "random" numbers will be the same - and it's exactly the thing that we want to avoid, to reduce contention.